app/models/delivery_attempt.rb

```diff
@@ -1,7 +1,7 @@
 class DeliveryAttempt < ApplicationRecord
   belongs_to :email
 
-  enum status: { sending: 0, delivered: 1, permanent_failure: 2, temporary_failure: 3, technical_failure: 4, internal_failure: 5 }
+  enum status: { sending: 0, delivered: 1, undeliverable_failure: 3, provider_communication_failure: 4 }
```

Comment:
Yup that's something I can do after this is merged. It can't be done as part of this PR though as the migration needs to run after this code is running.